### PR TITLE
Update Security and Privacy Considerations (2)

### PR DESCRIPTION
**Pull request by mmccool**
Resolves https://github.com/w3c/wot-architecture/issues/672

WIP: Do not merge, still being reviewed by Security TF

- add trusted environment risk

***
<a href="https://pr-preview.s3.amazonaws.com/mmccool/wot-architecture/pull/686.html" title="Last updated on Feb 2, 2022, 5:37 PM UTC (524e6bb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-architecture/686/7368123...mmccool:524e6bb.html" title="Last updated on Feb 2, 2022, 5:37 PM UTC (524e6bb)">Diff</a>

### CODE REVIEW

**Comment by mmccool** [comment]
To do: fix cross-references, href syntax is wrong, should set it up also so section names and numbers get filled in by respec.

**Comment by Citrullin** [comment]
I am fine with this. Well written.

**Review by Citrullin — APPROVED** [review]

**Review by j1y3p4rk — APPROVED** [review]
I am also fine with the changes.

File: index.html [hunk 1]
```diff
@@ -1158,6 +1160,14 @@ <h1>System Topologies (Horizontals)</h1>
       They are not exhaustive and are presented to motivate for the WoT architecture and building blocks
       that are defined in later sections of this specification.
     </p>
+    <p>
+    This section also makes use of the concept of a Trusted Environment,
```
Comment (mlagally) [inline index.html:1164]: suggest to create a terminology entry for "Trusted Environment"

**Pull request merged**